### ISSUE TITLE
BlockLength cop: ignore it by default?

### ISSUE THREAD

**Issue by tagliala** [sds/slim-lint#49]
Hi, what do you think about ignoring the above cop by default? It is noisy for slim templates and it doesn't report the proper number of lines

e.g.:

``` slim
.table-responsive
  table.table.table-striped
    tbody
      - elements.each do |element|
        tr
          td= link_to element
          - if element.editable?
            td.btn-toolbar
              = link_to edit_element_path(itinerary), class: 'btn btn-xs btn-default' do
                span.fa.fa-pencil<> aria-hidden='true'
                = t('.edit')
              = link_to element_path(itinerary), method: :delete, data: { confirm: t('.confirm') }, class: 'btn btn-xs btn-danger' do
                span.fa.fa-trash<> aria-hidden='true'
                = t('.destroy')
          - else
            td= t('.read_only')
```

Output:

```
_table.html.slim:4 [W] RuboCop: Block has too many lines. [28/25]
```
---

Current workaround:

``` yml
linters:
  RuboCop:
    ignored_cops:
      - Lint/BlockAlignment
      - Lint/EndAlignment
      - Lint/Void
      - Metrics/BlockLength # TODO: https://github.com/sds/slim-lint/issues/49
      - Metrics/LineLength
      - Style/AlignHash
      - Style/AlignParameters
      - Style/BlockNesting
      - Style/FileName
      - Style/FirstParameterIndentation
      - Style/FrozenStringLiteralComment
      - Style/IfUnlessModifier
      - Style/IndentationConsistency
      - Style/IndentationWidth
      - Style/MultilineArrayBraceLayout
      - Style/MultilineAssignmentLayout
      - Style/MultilineHashBraceLayout
      - Style/MultilineMethodCallBraceLayout
      - Style/MultilineMethodDefinitionBraceLayout
      - Style/MultilineMethodCallIndentation
      - Style/MultilineOperationIndentation
      - Style/Next
      - Style/TrailingBlankLines
      - Style/TrailingWhitespace
      - Style/WhileUntilModifier
```

**Comment by tagliala** [comment]
Thanks!
